docs/advanced/template_extensions.rst

```diff
@@ -27,6 +27,6 @@ Please note that Cookiecutter will **not** install any dependencies on its own!
 As a user you need to make sure you have all the extensions installed, before
 running Cookiecutter on a template that requires custom Jinja2 extensions.
 
-.. _`Jinja2 extensions`: http://jinja2.readthedocs.io/en/latest/extensions.html#extensions
+.. _`Jinja2 extensions`: http://jinja.pocoo.org/docs/2.10/extensions/
```

Comment:
Would you mind changing this to `http://jinja.pocoo.org/docs/latest/extensions/` so that the documentation always points to the latest stable version of jinja?

Reply:
done :)